src/wrapper.jl

```diff
@@ -920,6 +1017,49 @@ function FT_GetStatus(ftHandle::FT_HANDLE)
 end
 
 
+"""
+    FT_SetEventNotification(ftHandle::FT_HANDLE, dwEventMask, pvArg)
+
+Sets conditions for event notification.
+
+An application can use this function to setup conditions which allow a thread to block 
+until one of the conditions is met. Typically, an application will create an event, 
+call this function, then block on the event. When the conditions are met, the event is set,
+and the application thread unblocked. `dwEventMask` is a bit-map that describes the events
+the application is interested in. `pvArg` is interpreted as the handle of an event which 
+has been created by the application. If one of the event conditions is met, the event is 
+set. If `dwEventMask=FT_EVENT_RXCHAR`, the event will be set when a character has been 
+received by the device. If `dwEventMask=FT_EVENT_MODEM_STATUS`, the event will be set when 
+a change in the modem signals has been detected by the device.
+If `dwEventMask=FT_EVENT_LINE_STATUS`, the event will be set when a change in the line 
+status has been detected by the device.
+"""
+function FT_SetEventNotification(ftHandle::FT_HANDLE, dwEventMask, pvArg)
+  @assert any(dwEventMask.==[FT_EVENT_RXCHAR,FT_EVENT_MODEM_STATUS,FT_EVENT_LINE_STATUS])
```

Comment:
Whilst Julia probably clever enough not to do so, this could create two arrays and reduce them. Better to evaluate if the input is in a set defined by a tuple, e.g.,

`@assert dwEventMask ∈ (FT_EVENT_RXCHAR, FT_EVENT.......`

Reply:
Or could be modified to be an enumeration, and thus this would be enforced up the chain.

Reply:
That makes so much more sense. Going the tuple route here, because I ran into some issues using an enum since I'm re-exporting a similar but differently named enum a level up. Feel free to enlighten me on your enum suggestion after my next PR though.